examples/audio/audio_empty.rs

```diff
@@ -0,0 +1,35 @@
+use bevy::prelude::*;
```

Comment:
Definitely a matter of preference, but I don't think we need an example just to illustrate the empty method. With that criteria we'd have thousands of examples.

Reply:
You could update one of the existing audio example, like audio_control, to exit once the sink is empty

Reply:
I just think its solving a problem that doesn't need to be solved. I think we should wait until the demand for such an example presents itself.